config.migration.sample.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dataDir": "/tmp/data",
+  "sshHost": "user@hostname",
+  "syncServer": "http://localhost:8000",
+  "migrationEndpoint": "/api/v2/store/:repoId"
```

Comment:
Create a new PR for the migrate-only endpoint, and then change this to use that new endpoint.